earthmover/earthmover.py

```diff
@@ -86,7 +89,7 @@ def __init__(self,
 
         # Prepare the output directory for destinations.
         self.state_configs['output_dir'] = os.path.expanduser(self.state_configs['output_dir'])
-        if not os.path.isdir(self.state_configs['output_dir']):
+        if not os.path.isdir(self.state_configs['output_dir']) and not skip_mkdir:
```

Comment:
Maybe instead of introducing another Earthmover init argument, we can create the output directory in `execute()`, since the only command that should create this directory is `run`.